ios/GitPoint.xcodeproj/project.pbxproj

```diff
@@ -623,6 +642,7 @@
 			isa = PBXGroup;
 			children = (
 				273F19201F1E1AD5004B4410 /* libCodePush.a */,
+				48F9DE9B1F6DB8C60064194C /* libCodePush.a */,
```

Comment:
@housseindjirdeh not sure why Xcode added this line, you may want to have a closer look